src/web/layouts/lib/stickiness.tsx

```diff
@@ -1,30 +1,29 @@
 import { css } from '@emotion/react';
 
-import { border } from '@guardian/src-foundations/palette';
-
 import { getZIndex, getZIndexImportant } from '@frontend/web/lib/getZIndex';
 
 type Props = {
 	children?: React.ReactNode;
 };
 
-type StuckProps = Props & {
+type StuckProps = {
+	children?: React.ReactNode;
 	zIndex?: string;
 };
 
-// The advert is stuck to the top of the container as we scroll
-// until we hit the bottom of the wrapper that contains
-// the top banner and the header/navigation
-// We apply sticky positioning and z-indexes, the stickAdWrapper and headerWrapper
-// classes are tightly coupled.
```

Comment:
I deleted these comments because they stopped being true at some point and it's better to have no comments than incorrect ones